packages/core/src/odata/common/request/odata-request.ts

```diff
@@ -152,19 +152,11 @@ export class ODataRequest<RequestConfigT extends ODataRequestConfig> {
    *
    * @returns Key-value pairs where the key is the name of a header property and the value is the respective value
    */
-  async headers(): Promise<Record<string, string>> {
+  async headers(): Promise<Record<string, any>> {
```

Comment:
is this correct. Shouldn't headers not be a string to string map?

Reply:
Theoretically it could be anything and if I checked correctly, everywhere else we use any as well.